gnomad_tgps.py

```diff
@@ -0,0 +1,86 @@
+import hail as hl
+from gnomad_qc.v3.resources import basics
+import hailtop.fs as hfs
+
+# initialize Hail
+hl.init(log='test.log', default_reference='GRCh38', backend='spark')
+
+# get the site table to extract AC and AF frequencies
+gnomad_freq_path = "gs://gcp-public-data--gnomad/release/3.1.2/ht/genomes/gnomad.genomes.v3.1.2.sites.ht"
+gnomad_freq_ht = hl.read_table(gnomad_freq_path)
+
+# get gnomad V3 data and split multi-allelic variants
+vds = basics.get_gnomad_v3_vds()
+vds_mt = vds.variant_data
+vds_split = hl.experimental.sparse_split_multi(vds_mt)
+
+# get the metadata of gnomad v3
+meta_path = "gs://gnomad/metadata/genomes_v3.1/gnomad_v3.1_sample_qc_metadata.ht"
+meta_dat = hl.read_table(meta_path)
+
+# filter out unreleased samples
+release_ht = meta_dat.filter(meta_dat.release == True)
+vds_split_filterd = vds_split.filter_cols(hl.is_defined(release_ht[vds_split.col_key]))
```

Comment:
also, it might be helpful to do a quick `count_cols()` here to verify you're keeping the correct number of samples (you can save the number of samples in v3 as a constant and compare the output of counting columns to this constant)